docs/source/get-started/package-managers.rst

```diff
@@ -6,19 +6,100 @@ Use your favorite package manager to install Kokkos.
 System package managers
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-**DNF**
+DNF
+---
+
 You may use the Fedora Project package manager to install Kokkos
 https://packages.fedoraproject.org/pkgs/kokkos/
 
 Other package managers
 ~~~~~~~~~~~~~~~~~~~~~~
 
-`Spack <https://spack.io>`
---------------------------
+`Spack <https://spack.io>`_
+---------------------------
+
 Spack is a popular package manager for HPC.  Spack comes with installation recipes for Kokkos.
-https://packages.spack.io/package.html?name=kokkos
+
+The `Kokkos recipe webpage <https://packages.spack.io/package.html?name=kokkos>`_ summarizes the available versions of Kokkos
+and their options.
+
+Most of the time, Spack Kokkos' variants follow the same options as the Kokkos `CMake options <./configuration-guide.html>`_.
+List of available variants can be found by running
+
+.. code-block::
+
+    spack info kokkos
+
+
+Note that CPU micro-architectures are automatically inherited from Spack's configuration.
+
+Installing Kokkos with Spack
+++++++++++++++++++++++++++++
+
+To install Kokkos with Spack with default options, run:
 
 .. code-block::
 
     spack install kokkos
 
+
+To install Kokkos with Cuda backend enabled, run:
+
+.. code-block::
+
+    spack install kokkos +cuda cuda_arch=86
+
+
+Note that the `cuda_arch` option is specific to the target GPU architecture.  Here, the `cuda_arch` value `86` corresponds
+to the NVIDIA Ampere architecture. With Spack, the architecture must be specified explicitly (no auto-detection).
```

Comment:
Is there similar advice we should give for HIP?
Users might be confused by the fact Spack named the variant `rocm`.
What about SYCL?

Reply:
It is true that `rocm` instead of `hip` is not straightforward. I will add examples for AMD and Intel.